src/app/user/api/tokens/route.ts

```diff
@@ -0,0 +1,54 @@
+import { publicClient, transformMulticallResults } from '@/lib/viemPublicClient'
+import { tokenContracts } from '@/lib/contracts'
+import { RIFTokenAbi } from '@/lib/abis/RIFTokenAbi'
+import { Address } from 'viem'
+
+export interface TokenInfo {
+  symbol?: string
+  name?: string
+  decimals?: string
+}
+
+const tokensToQuery = {
+  [tokenContracts.RIF]: tokenContracts.RIF,
+  [tokenContracts.stRIF]: tokenContracts.stRIF,
+}
+
+const tokenAddresses = Object.values(tokensToQuery) as Address[]
```

Comment:
Why not simply `const tokenAddresses = [tokenContracts.RIF, tokenContracts.stRIF]`?

Reply:
If in the future we want to add more tokens, we simply have to modify tokensToQuery
We can also have it as an array but I find it's readable if we use an object map

Reply:
Yes but check it again,`tokensToQuery` is an map of `address: address`. It's redundant.

Reply:
I've removed the object map and added a comment instead